lib/active_merchant/billing/gateways/rapyd.rb

```diff
@@ -11,6 +11,8 @@ class RapydGateway < Gateway
       self.homepage_url = 'https://www.rapyd.net/'
       self.display_name = 'Rapyd Gateway'
 
+      USA_PAYMENT_METHODS = %w[us_debit_discover_card us_debit_mastercard_card us_debit_visa_card us_ach_bank]
```

Comment:
am curious about this change, What is the purpose of this line? I have not seen where it is used.

Reply:
Ohm you are right I forgot to push my last changes, it is used to validate if is a US PMT to include customer object data, Thank you